local_docs/fluid.buf2list.maxref.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes'?>
+<?xml-stylesheet href="./_c74_ref.xsl" type="text/xsl"?>
+<!--
+Part of the Fluid Corpus Manipulation Project (http://www.flucoma.org/)
+Copyright 2017-2019 University of Huddersfield.
+Licensed under the BSD-3 License.
+See license.md file in the project root for full license information.
+This project has received funding from the European Research Council (ERC)
+under the European Union’s Horizon 2020 research and innovation programme
+(grant agreement No 725899).
+-->
+<c74object name='fluid.buf2list' category='FluidCorpusManuipulation'>
+    <digest>Copy the contents of a buffer~ to a list</digest>
+    <description>A simplified workflow for extracting numerical data from buffers as lists of numbers.</description>
+    <discussion>
+        <h4><openfilelink filename="Fluid Corpus Manipulation Toolkit.maxpat">Open the Overview Patch</openfilelink></h4>
+        Given a source buffer that is valid, fluid.buf2list copies its contents to a new list when it receives a bang.
+
+This is a convenient way of extracting data from a buffer to be used elsewhere.
+
+        <p>fluid.buf2list is part of the Fluid Decomposition Toolkit of the FluCoMa project. For more explanations, learning material, and discussions on its musicianly uses, visit <a href="http://www.flucoma.org/">flucoma.org</a>.</p>
+    </discussion>
+	<!--METADATA-->
+	<metadatalist>
+		<metadata name='author'>FluCoMa</metadata>
+		<metadata name='tag'>Fluid Decomposition</metadata>
+		<metadata name='tag'>corpus</metadata>
+	</metadatalist>
+
+    <inletlist>
+    </inletlist>
+
+    <outletlist>
+    </outletlist>
+
+    <!--MESSAGES-->
+    <methodlist>
```

Comment:
There's also a `buffer` message. Check the redundant xml I sent you just to see if we agree on the content?

Reply:
Ah yes, so if a buffer object render that? I'll check the text in due course.